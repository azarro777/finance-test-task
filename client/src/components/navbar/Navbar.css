.navbar {
  display: flex;
  height: 50px;
  justify-content: center;
  background-color: white;
  width: 100%;
	box-shadow: 0 1px 2px hsla(0,0%,0%,0.05), 0 1px 4px hsla(0, 0%, 0%, 0.05), 0 2px 8px hsla(0, 0%, 0%, 0.05);
}
.navbar__header {
  margin-left: 10px;
  font-size: 24px;
  font-weight: 700;
}

.container {
  width: 1140px;
  display: flex;
  align-items: center;
}

.navbar__logo {
	width: 40px;
}