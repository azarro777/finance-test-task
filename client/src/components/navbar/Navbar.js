import './Navbar.css';
import logo from '../../assets/img/bars-business-and-finance.svg';

export const Navbar = () => {

	return (
		<div className="navbar">
			<div className="container">
				<img src={logo} alt="logo" className="navbar__logo"/>
				<div className="navbar__header">Finance</div>
			</div>
		</div>
	);
};