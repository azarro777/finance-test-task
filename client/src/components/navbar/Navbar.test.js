import React from 'react';
import { Navbar } from './Navbar';
import { shallow } from 'enzyme';

describe('Testing <Navbar/>', () => {
	it('Navbar have rendered correctly', () => {
		const navbar = shallow(<Navbar/>);
		expect(navbar).toMatchSnapshot();
	});
});