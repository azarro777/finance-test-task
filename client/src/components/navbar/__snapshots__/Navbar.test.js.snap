// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Navbar/> Navbar have rendered correctly 1`] = `ShallowWrapper {}`;
